packages/pigeon/CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+# Pigeon Contributor's Guide
+
+## Description
+
+Pigeon is a code generation tool that adds type safety to Flutter’s Platform
+Channels.  This document serves as an overview of how it functions to help
+people who would like to contribute to the project.
+
+## State Diagram
+
+Pigeon generates a temporary file in its LaunchIsolate, then launches another
```

Comment:
You might have to define these words for contributors.

Reply:
I defined LaunchIsolate and PigeonIsolate.